packages/@lwc/shared/src/html-attributes.ts

```diff
@@ -49,24 +49,36 @@ export function isBooleanAttribute(attrName: string, tagName: string): boolean {
     );
 }
 
+// This list is based on https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
 const GLOBAL_ATTRIBUTE = new Set([
-    'role',
```

Comment:
I removed `role` because it is not in [MDN's list of global attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).

This change should not be observable, because both `@lwc/template-compiler` and `@lwc/engine-server` were _also_ checking `isAriaAttribute`, and it turns out that `role` is an ARIA attribute.

https://github.com/salesforce/lwc/blob/dba53e754cf30e3ecad9fb150db8827d84140827/packages/%40lwc/template-compiler/src/parser/attribute.ts#L219-L220

https://github.com/salesforce/lwc/blob/2ddff92f4cb7c223e63a1ad791ae6b9bf2d66a42/packages/%40lwc/engine-server/src/renderer.ts#L170

https://github.com/salesforce/lwc/blob/d93e2963d8d32b138ce9ed130d98e7bef84b3d74/packages/%40lwc/shared/src/aria.ts#L66